src/windows/volume.cc

```diff
@@ -18,6 +18,32 @@
 
 static const size_t kVolumePathShortLength = 4;
 
+HRESULT drivelist::volume::GetSystemVolume(wchar_t *out) {
+  const char * const systemDrive = getenv("SystemDrive");
```

Comment:
I implemented the suggestion from https://github.com/resin-io-modules/drivelist/issues/126#issuecomment-312071764